x/axelarnet/handler.go

```diff
@@ -20,7 +21,7 @@ func NewHandler(k types.BaseKeeper, n types.Nexus, b types.BankKeeper, t types.I
 			res, err := server.Link(sdk.WrapSDKContext(ctx), msg)
 			result, err := sdk.WrapServiceResult(ctx, res, err)
 			if err == nil {
-				result.Log = fmt.Sprintf("successfully linked {%s} and {%s}", res.DepositAddr, msg.RecipientAddr)
+				result.Log = fmt.Sprintf("successfully linked deposit %s on %s to recipient %s on %s for %s", res.DepositAddr, exported.Axelarnet.Name, msg.RecipientAddr, msg.RecipientChain, msg.Asset)
```

Comment:
If we are spending such effort to enrich the logging info, why not move everything into tags/labels which is what the team decided to be better previously?

Reply:
There needs to be a big logging refactor based on the logging discussion. But until then, we would still like to improve it for debugging.

Reply:
Are we able to provide labels to `result.Log` - these are distinct from logging output in that they are part of the response for a successful tx, no?

Reply:
Although events are also included in the tx receipt, so perhaps string formatting in `result.Log` is also unnecessary

Reply:
Yeah, I'll log it separately in the Link request.